lib/instrumentation/openai.js

```diff
@@ -173,7 +184,14 @@ function recordChatCompletionMessages({ agent, segment, request, response, err }
  * messages
  *
  */
-function instrumentStream({ shim, request, response, segment }) {
+function instrumentStream({ agent, shim, request, response, segment }) {
+  if (!agent.config.ai_monitoring.streaming.enabled) {
+    shim.logger.warn(
+      '`ai_monitoring.streaming.enabled` is set to `false`, stream will not be instrumented.'
```

Comment:
we talked in an agent sync. we're going to log a supportability metric when streaming is disabled.  I will add here